src/aktualizr_secondary/aktualizr_secondary.h

```diff
@@ -21,11 +21,13 @@ class AktualizrSecondary : public Uptane::SecondaryInterface {
   AktualizrSecondary(AktualizrSecondaryConfig config, std::shared_ptr<INvStorage> storage,
                      std::shared_ptr<KeyManager> key_mngr, std::shared_ptr<UpdateAgent> update_agent);
 
+  std::string Type() const override { return ""; }
```

Comment:
What is the significance of leaving this empty by default?

Reply:
This is used on the secondary side and the `Type()` is not used there. But maybe setting it to  `ip` here would be more accurate.

Reply:
Sounds like it's fine as is if it isn't actually used. Is it just there because of the interface, then?

At some point, `aktualizr-secondary` was intended to be reusable by multiple communication protocols. How explicitly tied to the IP Secondary logic is it now?

Reply:
Ah I forgot about this, yes the method is here because of the interface.
I'll keep it as empty then.